Xamarin.Forms.Material.iOS/MaterialEntryRenderer.cs

```diff
@@ -11,6 +12,11 @@ public class MaterialEntryRenderer : EntryRendererBase<MaterialTextField>, IMate
 		protected override void UpdateColor() => Control?.UpdateTextColor(this);
 		protected virtual void ApplyTheme() => Control?.ApplyTheme(this);
 		protected override void UpdatePlaceholder() => Control?.UpdatePlaceholder(this);
+		protected override void UpdateAttributedPlaceHolder(NSAttributedString nSAttributedString)
```

Comment:
The casing in `nSAttributedString` seems a bit odd. Can we try `nsAttributedString`? here and in other places?